library/common/stats/engine_utility.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// From upstream Envoy
+#include "common/stats/utility.h"
+
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Stats {
+namespace Engine {
+namespace Utility {
```

Comment:
```
namespace Envoy {
namespace Stats {
namespace Utility {
```
yields a conflict from the `@envoy/source/common/stats/utility.h`, therefore added the `Engine` namespace here

Reply:
https://github.com/envoyproxy/envoy/pull/15603 will allow us to add the function without adding the Engine namespace. Will update as soon as that lands.